android/app/src/main/AndroidManifest.xml

```diff
@@ -144,6 +144,46 @@
                     android:path="/givt"
                     android:scheme="https" />
             </intent-filter>
+            <intent-filter android:autoVerify="true">
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+                <data android:scheme="givtkids" />
+            </intent-filter>
+
+            <intent-filter android:autoVerify="true">
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+                <data
+                    android:host="www.givt.app"
+                    android:path="/search-for-coin"
+                    android:scheme="https"/>
+
+            </intent-filter>
+
+            <intent-filter android:label="testDeepLink"
```

Comment:
I know you probably copied it, but this looks the same as above, except that this one has a label.